test/Window/Cursor.test.cpp

```diff
@@ -1,8 +1,36 @@
 #include <SFML/Window/Cursor.hpp>
 
+#include <doctest/doctest.h>
+
+#include <WindowUtil.hpp>
 #include <type_traits>
 
 static_assert(!std::is_copy_constructible_v<sf::Cursor>);
 static_assert(!std::is_copy_assignable_v<sf::Cursor>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::Cursor>);
 static_assert(!std::is_nothrow_move_assignable_v<sf::Cursor>);
+
+TEST_CASE("[Window] sf::Cursor" * doctest::skip(skipDisplayTests))
+{
+    SUBCASE("Construction")
+    {
+        const sf::Cursor cursor;
+    }
+
+    SUBCASE("loadFromPixels()")
+    {
+        sf::Cursor   cursor;
+        std::uint8_t pixels[4];
```

Comment:
Doesn't this array need some initialization?
What about using `std::array`? 🤔

Reply:
I figured it was fine letting these array elements be junk since we’re not testing anything about the particular pixel values. I can switch to std::array and initialize them if you want.

Reply:
Tests can be considered as sort of documentation, so I think it makes sense to initialize stuff and use modern constructs 🙂